src/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs

```diff
@@ -1414,7 +1414,7 @@ private static Task<Int32> Choose(CancellationToken cancellationToken)
             //var boxedCompleted = new StrongBox<Task>(); // Acts as both completion marker and sync obj for targets
             var result = new TaskCompletionSource<int>();
 
-            // Set up teardown cancellation.  We will request cancellation when a) the supplied options token
+            // Set up tear-down cancellation.  We will request cancellation when a) the supplied options token
```

Comment:
Yes, that was overly picky of me; *teardown* is perfectly cromulent as the joke goes. I'll revert those.